src/chapters/angular-wk1/introduction.rst

```diff
@@ -0,0 +1,22 @@
+Why Use JavaScript Libraries
+=============================
+
+We use libraries to make our lives as developers easier. We can focus on solving our specific problems if we don't have to spend time figuring out how information flows throughout our project.
+
+When it comes to web based applications, there are two very different places code can exist: in the user's browser (front end) and on the host's server (back end).
+
+For web app development, consider the front end as what the user interacts with and sees, while the back end contains the logic and manipulations that the user doesn't need to worry about. Similar to how an old mechanical clock works. The front end would be the face with the 12 numbers, and the two moving hands representing hours, and minutes. The user only needs the clock face to determine the time. The back end for the clock would be the various cogs, wheels, and power of the clock.
```

Comment:
Missed this before: Comma corrections. Remove them from the clock sentence, "The front end would be the face with the 12 numbers and the two moving hands representing hours and minutes."

Also, "power source" sounds better than "power of the clock", since it avoids using 'clock' twice in the sentence.